edvart/report.py

```diff
@@ -143,7 +144,7 @@ def export_notebook(
         notebook_filepath: Union[str, os.PathLike],
         dataset_name: str = "[INSERT DATASET NAME]",
         dataset_description: str = "[INSERT DATASET DESCRIPTION]",
-        export_data_mode: ExportDataMode = ExportDataMode.NONE,
+        export_data_mode: ExportDataMode = ExportDataMode.NONE,  # type: ignore
```

Comment:
Why the ignore?

I'd consider adding explanatory comment if it's not something obvious that I am missing.

Reply:
Added a comment.